core-web/libs/template-builder/src/lib/components/template-builder/utils/mocks.ts

```diff
@@ -288,7 +288,9 @@ export const MESSAGES_MOCK = {
     'editpage.layout.theme.search': 'Search',
     'dot.template.builder.classes.dialog.update.button': 'Update',
     'dot.template.builder.sidebar.header.title': 'Sidebar',
-    'dot.template.builder.row.box.wont.fit': 'Minimum 1 column needed for box drop.'
+    'dot.template.builder.row.box.wont.fit': 'Minimum 1 column needed for box drop.',
+    'dot.template.builder.autocomplete.has.suggestions': 'Add the message for suggestions case',
+    'dot.template.builder.autocomplete.no.suggestions': 'Add the message for no suggestions case'
```

Comment:
Update this messages for the story and also add the missing ones

Reply:
You can use this object in the test for the messages.

Reply:
Not needed for tests... overkill